state_manager/state_manager/messages.py

```diff
@@ -26,15 +26,16 @@
 class Response:
     """A socket response."""
 
-    content: Optional[str]
+    content: Optional[bytes]
     is_error: bool = False
+    broadcast: bool = False
 
     def to_bytes(self) -> bytes:
         """Convert Response object into bytes."""
         if self.content is None:
             return b""
-        message = f"ERROR: {self.content}" if self.is_error else self.content
-        return message.encode()
+        message = b"ERROR: " + self.content if self.is_error else self.content
```

Comment:
kinda prefer the old way here - the `+` makes it a little tougher to parse what the `if` applies to, i just like f-strings a lot, etc